### PR TITLE
[INLONG-6238][Doc] Add configuration doc for DataProxy

### PR DESCRIPTION
**Pull request by gosonzhang**
- Fixes : https://github.com/apache/inlong/issues/6238

### CODE REVIEW

File: docs/modules/dataproxy/configuration.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+| prometheusHttpPort                                      | The port when reporting using Prometheus                                                           | 9081                               |                                                                                                                   |
+| audit.enable                                            | Whether to enable data reporting InLong-Audit service                                                         | true                                |                                                                                                                   |
+| audit.proxys                                            | The address of the InLong-Audit service                                                                | 127.0.0.1:10081                      | 格式按照 {ip1:port1}[ {ip2:port2}][ {ip3:port3}]进行配置，不同ip:port间通过空格、回车，或换行符进行分隔                                       |
```
Comment (healchow) [inline docs/modules/dataproxy/configuration.md:21]: Should change to English.

Comment (gosonzhang) [reply docs/modules/dataproxy/configuration.md]: Ok!

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/dataproxy/configuration.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: 配置
+sidebar_position: 3
+---
+
+## 基本配置（common.properties）
+
+| 参数                                                      | 描述                       | 默认值                                  | 备注                                                                          |
+|---------------------------------------------------------|--------------------------|--------------------------------------|-----------------------------------------------------------------------------|
+| manager.hosts                                           | Manager服务器IP和端口列表        | 127.0.0.1:8083                      | 不允许为空，格式按照 {ip1:port1}[,{ip2:port2}][,{ip3:port3}] 格式进行配置                   |
```
Comment (healchow) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/dataproxy/configuration.md:10]: Suggest adding one blank before or between English and Chinese.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/dataproxy/configuration.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+| manager.auth.secretId                                   | 登录所需帐号                   | 无                                    | 允许为空不填写                                                                     |
+| manager.auth.secretKey                                  | 登录所需密码                   | 无                                    | 允许为空不填写                                                                     |
+| proxy.cluster.tag                                       | dataproxy所处的集群Set名       | default_cluster                      | 一个集群set里可以包含多个dataproxy、MQ的集群                                               |
```
Comment (healchow) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/dataproxy/configuration.md:13]: `dataproxy所处的集群Set名` -> `DataProxy 所处的集群 Tag 名`.

`Set` is a new name, suggest unifying it to `Tag`.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/dataproxy/configuration.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+DataProxy 支持配置化的 source-channel-sink，配置方式与 flume 的配置文件结构类似。配置文件放在 dataproxy-*.conf 文件中，目前支持 dataproxy-pulsar.conf 和 dataproxy-tube.conf 两种，用于区分不同的中间件类型，具体类型可以在启动时指定，默认（未指定时）使用 dataproxy-pulsar.conf 作为配置文件。
+
+
+## 其他自动更新配置
+DataProxy启动并成功链接到Manager后，会自动从Manager同步运行配置，并定期更新，如下部分配置不可修改
+
+| 配置文件名            | 描述                    |  备注             |
+|------------------|-----------------------|-----------------|
+| mq_cluster.properties         | MQ集群配置                |     |
+| topics.properties  | groupId到MQ的Topic的映射配置 |     |
```
Comment (healchow) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/dataproxy/configuration.md:51]: `groupId到MQ的Topic的映射配置` -> `InlongGroupId 和不同类型 MQ 的 Topic 的映射配置`.

**Review by healchow — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**